src/main/initNrfUtilProxyEnv.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import argv from './argv';
+
+export default () => {
+    const proxyServer = argv['proxy-server'] as string | undefined;
+    const httpProxy = proxyServer
+        ?.split(';')
+        .find(p => p.startsWith('http='))
+        ?.split('=')[1];
```

Comment:
This now supports it if customers use a command line option like `--proxy-server="http=foopy:80;ftp=foopy2"` but it doesn't support the (IMO more important) options like `--proxy-server="foopy:8080"`. And it is also not supported if `https` and `http` are both specified (e.g. `--proxy-server="http=foopy:80;https=foopy2"`). 

For now I suggest to just support `--proxy-server="foopy:8080"` by using `proxyServer` directly instead of `httpProxy`.